.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: python
+env:
+   - DJANGO='>=1.8,<1.9'
+   - DJANGO='>=1.9,<1.10'
+   - DJANGO='>=1.10,<1.11'
+   - DJANGO='>=1.11,<1.12'
```

Comment:
if we group them all together as `>=1.8,<1.12` does it run the suite for all minor versions too?